k8s-install/README.md

```diff
@@ -2,7 +2,7 @@
 
 This directory includes manifests for deploying canal on Kubernetes using the Kubernetes API.  
 
-#### For Kubernetes 1.7
+#### For Kubernetes 1.7 and 1.8
```

Comment:
Perhaps "Kubernetes 1.7.+"?

v1.9. is just around the corner.

Reply:
I think I'd prefer being explicit and saying `For Kubernetes 1.7, 1.8, and 1.9`, that way users aren't wondering, when was the last time was this updated (what was out when this was updated)?

Reply:
WDYT?

Reply:
SGTM. Make sure you remember to update when v1.9 is released.